test/functional/api/v1/users_controller_test.rb

```diff
@@ -137,22 +137,20 @@ def user_one_as_anonymous_viewer
   end
 
 # do we support this?
-=begin
-  test "should recreate the admin account" do
-    user = users(:one)
-    user.update_attribute :admin, true
-
-    User.find_by_login("admin").delete # Of course we only use destroy in the codebase
-    assert User.find_by_login("admin").nil?
-
-    as_user :one do
-      get :index, {}
-      assert_response :success
-    end
-
-    assert !User.find_by_login("admin").nil?
-  end
-=end
+#   test "should recreate the admin account" do
```

Comment:
Would you mind deleting this block?  I can categorically state that we do not any more :)